src/steamship/data/operations/generator.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from typing import List, Optional
+
+from steamship.base.request import Request
+from steamship.base.response import Response
+from steamship.data.block import Block
+
+
+class GenerateRequest(Request):
+    """This class provides the input for a request to a Generator.  There are several ways to specify the input; see below"""
+
+    # Input Specification
+
+    # Must specify plugin instance to use
+    plugin_instance: str = None
```

Comment:
clarity: this is instance handle, correct?

Reply:
Yeah, I copied the behavior of tag, which is awkward. Can change here if the inconsistency is better than the bad-naming